src/server.js

```diff
@@ -9,57 +9,96 @@ const app = express();
 
 const {getLogTransportConsole, getLogTransportFile, getLogResolvePath} = require('./lib/logFormat');
 const ConstructURL = require('./utils/ConstructURL');
+const ContentIndexer = require('./utils/lunr/ContentIndexer');
+const SearchHandler = require('./utils/lunr/SearchHandler');
 const { validate } = new Validator();
+let message = 'Loading ...';
 
+// Method to format the writings logs
+getLogTransportConsole();
+getLogTransportFile();
+getLogResolvePath();
 
-window.addEventListener('DOMContentLoaded', () => {
-    // Method to format the writings logs
-    getLogTransportConsole();
-    getLogTransportFile();
-    getLogResolvePath();
+log.info('Server started');
 
-    log.info('Server started');
+app
+.use(cors())
+.use(bodyParser.json())
+.use(express.static(path.join(__dirname, '../static')));
 
-    app
-    .use(cors())
-    .use(bodyParser.json())
-    .use(express.static(path.join(__dirname, '../static')))
+ContentIndexer.tocIndexer();
```

Comment:
If the documentation is not inside the desktop viewer, it will crash. Do it only if the option is enable.